webui/react/src/components/Section.tsx

```diff
@@ -40,12 +42,15 @@ const Section: React.FC<Props> = (props: PropsWithChildren<Props>) => {
           {props.options && <div className={css.options}>{props.options}</div>}
         </div>
       )}
-      {props.filters && (
+      {!props.loading && props.filters && (
         <div className={css.filterBar}>
           {props.filters}
         </div>
       )}
-      <div className={css.body}>{props.children}</div>
+      <div className={css.body}>
+        {!props.loading && props.children}
+        {props.loading && <Spinner />}
```

Comment:
love that, super-minor, but looks to be not centered?

<img width="893" alt="Schermata 2021-03-26 alle 09 39 27" src="https://user-images.githubusercontent.com/5413702/112605067-2d47bc00-8e17-11eb-85bf-70e67a206624.png">

Reply:
good eye! Should be good to go now.